ingestion/src/metadata/cmd.py

```diff
@@ -282,6 +282,13 @@ def docker(
     default=None,
     required=False,
 )
+@click.option(
+    "--uploadDestinationType",
+    help="AWS or AZURE",
+    type=click.Choice(['AZURE', 'AWS'], case_sensitive=False),
```

Comment:
If we are not setting case sensitive here, the checks later (e.g., `uploadDestinationType == "AWS":`) will fail